### PR TITLE
feat: my feed analytics

### PR DESCRIPTION
**Pull request by rebelchris**
Introducing the analytics events for the My Feed, as well as sending which feed events come from.

DD-432 #done

### CODE REVIEW

**Review by sshanzel — APPROVED** [review]
As sharp as it can get! 🔥

**Review by ayadotdev — APPROVED** [review]
Looks OK to me 👍

**Review by idoshamun — CHANGES_REQUESTED** [review]
Another thing I'm missing here is tracking the actual filters when logged out. It's important for the funnel and also understanding what the user filters before they register. I guess this is already implemented for logged users

File: packages/shared/src/components/filters/FeedFilters.tsx [hunk 1]
```diff
@@ -29,14 +29,20 @@ export default function FeedFilters({
...
   const { hasAnyFilter } = useFeedSettings();
   const { flags } = useContext(FeaturesContext);
   const shouldShowMyFeed = isFeaturedEnabled(Features.MyFeedOn, flags);
 
   const onCreate = () => {
-    // apply analytics
+    trackEvent({
+      event_name: 'click',
+      target_type: 'create my feed button',
+      target_id: 'feed-filters',
```
[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/components/filters/FeedFilters.tsx:43]: Hmm the main thing is this should be different from the one we capture in the ProfileForm.
This is the click, and the other one is the actual registration.

[the remark this answers is not in the document]
Comment (idoshamun) [reply packages/shared/src/components/filters/FeedFilters.tsx:43]: It's just that feed filters is not explicit enough. We need to think of something else

File: packages/shared/src/components/filters/FeedFilters.tsx [hunk 2]
```diff
@@ -29,14 +29,20 @@ export default function FeedFilters({
...
+      feed_item_title: 'create feed filters',
```
[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/components/filters/FeedFilters.tsx:44]: We don't have AB on this button, it always says "Create" should we use that as the feed_item_title?

[the remark this answers is not in the document]
Comment (idoshamun) [reply packages/shared/src/components/filters/FeedFilters.tsx:44]: We can skip that for now

**Comment by rebelchris** [comment]
@idoshamun Refactored two events.
Think that should do it.

In regards to the filter events, they actually haven't changed to they still work the same way for anonymous users as for logged users.

File: packages/shared/src/components/sidebar/MyFeedButton.tsx [hunk 3]
```diff
@@ -66,7 +66,7 @@ const getAnalyticsEvent = (
 ): Partial<AnalyticsEvent> => ({
   event_name: eventName,
   target_type: 'my feed button',
-  target_id: 'sidebar',
+  target_id: 'my-feed',
```
Comment (idoshamun) [inline packages/shared/src/components/sidebar/MyFeedButton.tsx:69]: Actually sidebar was better, I think.
This is why I resolved my comment, didn't want you to change after our discussion

**Review by idoshamun — APPROVED** [review]
Approved but have a look at my comment before merging :)

File: packages/shared/src/components/sidebar/MyFeedButton.tsx (hunk 3 above)
Comment (rebelchris) [reply packages/shared/src/components/sidebar/MyFeedButton.tsx:69]: Oh sorry I already changed it yesterday, will revert this one :)

**Pull request merged**